addons/mail/static/src/js/services/mail_notification_manager.js

```diff
@@ -468,12 +468,11 @@ MailManager.include({
      *
      * @private
      * @param {Object} data
-     * @param {string} data.author_id
      */
     _handlePartnerTransientMessageNotification: function (data) {
         var lastMessage = _.last(this._messages);
         data.id = (lastMessage ? lastMessage.getID() : 0) + 0.01;
-        data.author_id = data.author_id || this.getOdoobotID();
+        data.author_id = this.getOdoobotID();
```

Comment:
`data.author_id` had value `"ODOOBOT"`, hence the removal here.
We want to enforce correct format for `author_id`: `["ODOOBOT", "ODOOBOT"]`.